### DISCUSSION TITLE
Removing a previous git account from your PC (Windows 10)

### DISCUSSION THREAD

**Discussion by ghost** [community/community#135577] [New to GitHub] [answered]
Hello,

I am using windows 10 on my pc. I have a previous gitlab account that I no longer want to use on my PC for various reasons. I want to remove said account from PC completely and I cant do it.

I tried going to Manage your credentials -> Web Credentials and removing a "git:https://gitlab...." file there but my PC still had this specific git profile . I also tried uninstalling all git related apps etc. This didint work either.

I also have a git remote repository on my pc relating to the old account.

I want to start completely new with nothing on my PC relating to my git profile. So I have 2 questions.
How do I uninstall/remove git completely from my pc so that I can do a clean install with a brand new account?
How do I remove just the user account, without deleting any other files, if I want to stop using that account in the future?

**Comment by everforyou** [comment, marked as answer]
- If you don't just want to see in your browser, please sign out in right navigation bar like below.
![image](https://github.com/user-attachments/assets/49a7a8cb-77b6-42a0-8e5b-f6712401ef78)

- If you want to delete your github account permanently, please delete your account in github setting.
![image](https://github.com/user-attachments/assets/7cd1c4fd-fc9f-44f2-9bff-b5e73f6c478e)
You can do this in here https://github.com/admin

- If you want to change existing local git repo to new account, you can use git bash command.
1. **Open Git Bash** and navigate to your local repository:

   ```bash
   cd /path/to/your/repo
   ```

2. **Check the current remote URL**:

   ```bash
   git remote -v
   ```

   This will show you the current remote repositories associated with your local repository.

3. **Set the new remote URL**:

   Replace `NEW_USERNAME` with your new GitHub username and `REPO_NAME` with the name of your repository:

   ```bash
   git remote set-url origin https://github.com/NEW_USERNAME/REPO_NAME.git
   ```

4. **Verify the change**:

   Run the following command again to ensure the remote URL has been updated:

   ```bash
   git remote -v
   ```
